ui/app/modifiers/code-mirror.ts

```diff
@@ -42,7 +42,7 @@ export default class CodeMirrorModifier extends Modifier<Args> {
     let editor = codemirror(this.element, {
       ..._PRESET_DEFAULTS,
       ...this.args.named.options,
-      value: this.args.named.value,
+      value: this.args.named.value ? this.args.named.value : '',
```

Comment:
[sand] I’d encourage use of `??` in cases like this, just cause it’s slightly easier to scan than a ternary.

```suggestion
      value: this.args.named.value ?? '',
```